File: QB_Challenges/PrepInsta/JAVA/01_Getting_Started/06_GO_Three_Nums/GOTNternary.java
import java.util.Scanner;

public class GOTNternary {

    public static void GOTN() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first name: ");
        int num1 = sc.nextInt();
        System.out.print("Enter the second name: ");
        int num2 = sc.nextInt();
        System.out.print("Enter the third name: ");
        int num3 = sc.nextInt();
        sc.close();

        int temp, result;
        
        temp = num1>num2 ? num1:num2;
        result = temp>num3 ? temp:num3;

        System.out.println(result + " is greatest number here");
    
    }

    public static void main(String[] args) {
        GOTN();
    }
    
}

File: QB_Challenges/PrepInsta/CPP/01_Getting_Started/06_GO_Three_Nums/GOTNternary.cpp
#include <iostream>

using namespace std;

void GOTN(){
    int num1, num2, num3;
    cout <<"Enter the first number: ";
    cin >>num1;
    cout <<"Enter the second number: ";
    cin >>num2;
    cout <<"Enter the third number: ";
    cin >>num3;

    int temp, result;

    temp = (num1 >= num2) ? num1 : num2;
    result = (temp >= num3) ? temp : num3;

    cout <<result <<" is the greatest number" <<endl;

}

int main(){
    GOTN();
    return 0;
}